cardano-cli/src/Cardano/CLI/Run/Debug/LogEpochState.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Cardano.CLI.Run.Debug.LogEpochState
+  ( runLogEpochStateCmd
+  ) where
+
+import           Cardano.Api
+import qualified Cardano.Api as Api
+
+import           Cardano.CLI.Commands.Debug.LogEpochState
+import           Cardano.CLI.Orphans ()
+
+import           Data.Aeson
+import qualified Data.ByteString.Lazy.Char8 as BSC
+import qualified System.IO as IO
+
+runLogEpochStateCmd
+  :: LogEpochStateCmdArgs
+  -> IO ()
+runLogEpochStateCmd
+    LogEpochStateCmdArgs
+      { nodeSocketPath
+      , configurationFile
+      , outputFilePath = File outputFilePath
+      } = do
+  appendFile outputFilePath ""
+
+  result <- runExceptT $ foldEpochState
+    configurationFile
+    nodeSocketPath
+    Api.QuickValidation
+    (EpochNo maxBound)
+    ()
+    (\(AnyNewEpochState sbe nes) _ _ -> do
+        liftIO $ appendFile outputFilePath $ BSC.unpack (shelleyBasedEraConstraints sbe $ encode nes) <> "\n"
```

Comment:
Use https://hackage.haskell.org/package/bytestring-0.12.1.0/docs/Data-ByteString.html#v:writeFile instead of `System.IO` one

Reply:
Done!